commit 8f0e631d2e59e3788af81da6edae5b3648d393ee (refs/changes/38/838/3)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2016-11-04T13:31:46+00:00 (2 years, 11 months ago)
    
    joyent/node-manta#280 honor options with job expressions
    Reviewed by: David Pacheco <dap@joyent.com>
    Reviewed by: Trent Mick <trent.mick@joyent.com>
    Approved by: Trent Mick <trent.mick@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index a1a936b..206b5bb 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,6 +2,8 @@
 
 ## not yet released
 
+- joyent/node-manta#280 honor options with job expressions
+
 ## 3.1.3
 
 - joyent/node-manta#277 mjob fails with "mjob: AssertionError: body (object) is required"
diff --git a/bin/mjob b/bin/mjob
index d08dfa7..e252c56 100755
--- a/bin/mjob
+++ b/bin/mjob
@@ -633,7 +633,10 @@ MJob.prototype.do_create = function create(subcmd, opts, args, cb) {
         function addPhase() {
             var p = {
                 exec: exec,
+                disk: opts.disk,
                 image: opts.image,
+                init: opts.init,
+                memory: opts.memory,
                 type: type
             };
 
